jax/_src/random.py

```diff
@@ -488,10 +488,12 @@ def randint(key: ArrayLike,
   dtypes.check_user_dtype_supported(dtype)
   dtype = dtypes.canonicalize_dtype(dtype)
   shape = core.canonicalize_shape(shape)
-  return _randint(key, shape, minval, maxval, dtype)
+  out_sharding = canonicalize_sharding(out_sharding, "randint")
+  return maybe_auto_axes(_randint, out_sharding, shape=shape, dtype=dtype)(
```

Comment:
Maybe format like this?

```
maybe_auto_axes(_randint, out_sharding, shape=shape,
                                dtype=dtype)(key, minval, maxval)
```

Reply:
ugh, this came out worse but instead of splitting of `(`, maybe bring `dtype=dtype` to the new line to it ends up looking better.